ci: remove podsecurity feature-gate

Madhu-1 opened the pull request:
remove the podsecurity feature-gate from minikube.sh, because of it kubernetes 1.25.0 deployment is failing

fixes: #3358

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

Madhu-1 committed:
ci: remove podsecurity feature-gate

remove the podsecurity feature-gate
from minikube.sh, because of it
kubernetes 1.25.0 deployment is failing

fixes: #3358

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

Madhu-1 commented:
CI might fail as we might need to fix some tests also.

Madhu-1 committed:
e2e: set securityContext of app pods

set required securityContext for app
pods for the e2e tests

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

nixpanic commented:
> CI might fail as we might need to fix some tests also.

Indeed, that seems to be the case.

Madhu-1 committed:
e2e: log checkAppMntSize output

logging checkAppMntSize output to
identify any problem with resize
operation.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

Madhu-1 committed:
e2e: log stdError in command execution fails

Along with err log stdError when the command
execution fails, this helps in debugging.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

Madhu-1 committed:
e2e: add FSGroup and RunAsUser to all pods

adding FSGroup and RunAsUser to all pods as
we dont want to run all pods as root user.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

nixpanic commented:
Superseded by #3362
